modules/network-observability-loki-install.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-loki-installation_{context}"]
+
+.Prerequisites
+
+* Installed Loki, recommended to install Loki using the Operator version 5.6, which allows the ability to specify a `network` tenant for Loki. The `network` tenant uses a specific tenant mode only implemented in version 5.6 of the Loki Operator. 
+* Object storage, such as AWS S3
+* {product-title} 4.10+
+* Linux Kernel 4.18
```

Comment:
sorry for the late comment about it, but it should be `4.18+` (higher versions are fine)

Reply:
Seriously no apologies needed :)